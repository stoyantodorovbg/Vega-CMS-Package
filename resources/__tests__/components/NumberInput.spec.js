import { mount } from "@vue/test-utils";
import NumberInput from "../../js/components/filters/NumberFilterInput";

describe('NumberInput Component', () => {
  test("should mount without crashing", () => {
    const wrapper = mount(NumberInput);

    expect(NumberInput).toMatchSnapshot();
  });
});
