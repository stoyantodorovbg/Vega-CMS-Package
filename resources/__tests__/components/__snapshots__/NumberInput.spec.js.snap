// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberInput Component should mount without crashing 1`] = `
Object {
  "$_vueTestUtils_original": [Circular],
  "_Ctor": Object {},
  "data": [Function],
  "methods": Object {
    "changeInput": [Function],
  },
  "name": "NumberInput",
  "props": Object {
    "name": Object {
      "default": "id",
      "type": [Function],
    },
  },
  "render": [Function],
  "staticRenderFns": Array [],
}
`;
